pkg/schedule/checker/checker_controller.go

```diff
@@ -181,6 +330,41 @@ func (c *Controller) RemovePriorityRegions(id uint64) {
 	c.priorityInspector.RemovePriorityRegion(id)
 }
 
+// CheckSuspectRanges would pop one suspect key range group
+// The regions of new version key range and old version key range would be placed into
+// the suspect regions map
+func (c *Controller) CheckSuspectRanges() {
```

Comment:
Do we need to check `if c == nil ` in `func (c *Coordinator) checkSuspectRanges() {`?

Reply:
It is always initialized in NewCoordinator

Reply:
Got it, just noticed the `c==nil` check in `GetPatrolRegionsDuration`. Seems like its an API called.